terraform/libvirt/outputs.tf

```diff
@@ -4,9 +4,9 @@ output "cluster_nodes_ip" {
   value = module.hana_node.output_data.private_addresses
 }
 
-output "cluster_nodes_public_ip" {
-  value = module.hana_node.output_data.addresses
-}
+#output "cluster_nodes_public_ip" {
```

Comment:
Is it not one of the important values needed to be exported in the Ansible inventory? Is it not relevant for libvrt?

Reply:
Yes it is but currently it is broken and blocking deployment to finish. Since libvirt is not a priority I left it out till it is fixed.